css/style.css

```diff
@@ -0,0 +1,84 @@
+/**
```

Comment:
Plan to use LESS on the final build - there's two types of sub-themes: CDN and LESS. CDN are really good for testing, overrides and quick changes so you rapidly prototype and LESS much better for overall management. Once it's stable plan to build a final subtheme in this way.